server/middleware/authenticate/index.js

```diff
@@ -0,0 +1,14 @@
+const jwt = require('jsonwebtoken');
+require('dotenv').config();
+
+export default function authenticateToken(req, res, next) {
+  const authHeader = req.headers.authorization;
+  const token = authHeader && authHeader.split(' ')[1];
+
+  if (!token) return res.json(401, { error: 'Token not found!' });
```

Comment:
this should be an array like the other GraphQL responses so that its consistent. 
```
{ errors: [`Token not found!`] }
```